certified-connectors/CobbleStone - Contract Insight/apiProperties.json

```diff
@@ -85,7 +85,7 @@
         "type": "string",
         "uiDefinition": {
           "constraints": {
-            "required": "true",
+            "required": "false",
```

Comment:
What is the purpose of this setting?

Reply:
this is not our code. just refreshing the code from Microsoft upstream.